src/CMakeLists.txt

```diff
@@ -149,6 +149,9 @@ if(IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/jit32")
 endif()
 
 add_subdirectory(gc)
```

Comment:
Do you still need the build the gc directory? The makefile in it is ugly because of it is referencing VM - I was not able to mirror it to corert because of it is causing build breaks there. It would be nice to get rid of it.